modules/navier_stokes/unit/src/main.C

```diff
@@ -0,0 +1,34 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "NavierStokesApp.h"
+#include "gtest/gtest.h"
+
+// Moose includes
+#include "Moose.h"
+#include "MooseInit.h"
+#include "AppFactory.h"
+
+#include <fstream>
+#include <string>
+
+PerfLog Moose::perf_log("gtest");
+
+GTEST_API_ int
+main(int argc, char ** argv)
+{
+  // gtest removes (only) its args from argc and argv - so this  must be before moose init
```

Comment:
```suggestion
  // gtest removes (only) its args from argc and argv - so this must be before moose init
```

Reply:
If this bothers you... go fix it everywhere in `modules/*/unit/src/main.C` 😆